src/codecs/cats/ldpc_matrices.h

```diff
@@ -0,0 +1,26 @@
+#ifndef __CATS_LDPC_MATRICES_H
+#define __CATS_LDPC_MATRICES_H
+
+typedef struct cats_ldpc_code {
+    // Code length (data+parity)
```

Comment:
Ok, now I see these comments. Could we just name the variables accordingly, so no comments would be necessary?